src/commands/generate/utils.ts

```diff
@@ -101,29 +102,47 @@ const strictModeReservedKeywords = new Set([
 ]);
 
 type DeprecatableDefinitions = IFieldDefinition | IEnumValueDefinition | IEndpointDefinition;
-export function addDeprecatedToDocs<T extends DeprecatableDefinitions>(typeDefintion: T): string[] | undefined {
-    if (typeDefintion.deprecated !== undefined && typeDefintion.deprecated !== null) {
-        if (typeDefintion.docs !== undefined && typeDefintion.docs !== null) {
+
+export function addDeprecatedToDocs<T extends DeprecatableDefinitions>(typeDefintion: T): string | undefined {
+    if (typeDefintion.deprecated != null && typeDefintion.deprecated != null) {
+        if (typeDefintion.docs != null && typeDefintion.docs != null) {
             // Do not add deprecated JSDoc if already exists
             if (typeDefintion.docs.indexOf("@deprecated") === -1) {
-                return [typeDefintion.docs + `\n@deprecated ${typeDefintion.deprecated}`];
+                return `${typeDefintion.docs}\n@deprecated ${typeDefintion.deprecated}`;
             }
         } else {
-            return [`@deprecated ${typeDefintion.deprecated}`];
+            return `@deprecated ${typeDefintion.deprecated}`;
         }
     }
-    return typeDefintion.docs != null ? [typeDefintion.docs] : undefined;
+    return typeDefintion.docs != null ? typeDefintion.docs : undefined;
 }
 
-export function addIncubatingDocs(
+export function addIncubatingToDocs(
     endpointDefinition: IEndpointDefinition,
-    existingDocs: string[] | undefined,
-): string[] | undefined {
-    if (endpointDefinition.tags !== undefined && endpointDefinition.tags.indexOf("incubating") >= 0) {
-        if (existingDocs === undefined) {
-            return [`@incubating`];
+    existingDocs: string | undefined,
+): string | undefined {
+    if (endpointDefinition.tags != null && endpointDefinition.tags.indexOf("incubating") >= 0) {
+        if (existingDocs == null) {
+            return "@incubating";
         } else {
-            return [existingDocs + `\n@incubating`];
+            return `${existingDocs}\n@incubating`;
+        }
+    }
+    return existingDocs;
+}
```

Comment:
Changed `existingDocs` and return types from `string[]` to `string` because this function will always either return an array of length 1 or `undefined`.

I believe the return typo is set to `string[] | undefined` because that's what the callers expect in typeGenerator.ts ([example](https://github.com/palantir/conjure-typescript/blob/f02cacb715a00dfa5eb3c819572924ae11d6876b/src/commands/generate/typeGenerator.ts#L127)). This should however be handled by the caller and not this function directly.

Additionally, leaving the code as it currently is will generate incorrectly formatted documentation if `addIncubatingToDocs` is called with `existingDocs` that has more than element.

For example,
```
addIncubatingToDocs({...}, ["docs_one", "docs_two"]) // returns ["docs_one,docs_two\nincubating"]
```

The same applies to `addErrorsToDocs` and `addDeprecatedToDocs`.